docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -0,0 +1,708 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "739993c7",
+   "metadata": {},
+   "source": [
+    "# Protein Folding"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d64f8162",
+   "metadata": {},
+   "source": [
+    "### Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bd209d1",
+   "metadata": {},
+   "source": [
+    "The structure and function of many natural and human-engineered\n",
+    "proteins is still only poorly understood. As a result, our understanding of\n",
+    "processes connected with protein folding, such as those encountered in\n",
+    "Alzheimer’s disease, vaccine development, and crop improvement\n",
+    "research, has remained limited.\n",
+    "\n",
+    "Unfolded polypeptides have a very large number of degrees of freedom\n",
+    "and thus an enormous number of potential conformations. For example, a\n",
+    "chain with 100 amino acids has on the order of 10 conformations. In\n",
```

Comment:
> For example, a chain with 100 amino acids has on the order of 10 conformations.

This seems like it should be a greater number of conformations...

Reply:
I missed the superscript! Thanks!